test/testref/dirac_1GridPoint.ref

```diff
@@ -0,0 +1,2 @@
+Test     : Input Dirac increment: min = 0, max = 1, mean = 3.08642e-05
```

Comment:
is this file being used for anything? if not, delte

Reply:
It's deleted.